tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -294,7 +297,7 @@ def is_gpu_available():
 
     local_device_protos = device_lib.list_local_devices()
     gpu_list = [x.name for x in local_device_protos if x.device_type == "GPU"]
-    if len(gpu_list) > 0:
+    if len(gpu_list) > 0:  # pylint: disable=len-as-condition
```

Comment:
given the list is initialized on previous like, i think you can adopt the preferred syntax `if gpu_list:` here

Reply:
Ok！